README.md

```diff
@@ -6,7 +6,7 @@ Regrettably, whenever you need it, you left it home. And if only, because it is
 
 Now you can use this small application to use it without needing to carry more than you already have - your computer.
 
-[![Timer](timer.png)](https://jschirrmacher.github.io/timer/?value=1200)
+[![Timer](timer.png)](https://rdmueller.github.io/timer/?duration=1200)
```

Comment:
When merging to this repo, we should change this back to the repo name

Reply:
yep. I wasn't sure if you would accept the merge.
I will not change it back and we can even remove my repo afterwards.

Reply:
done